README.md

```diff
@@ -7,11 +7,11 @@
 
 ![codecov.io](https://codecov.io/github/owncloud/contacts/branch.svg?branch=master)
 
-**A contacts app for [ownCloud](https://owncloud.org) and [Nextcloud](https://nextcloud.com).**
+**A contacts app for [Nextcloud](https://nextcloud.com) and [ownCloud](https://owncloud.org).**
```

Comment:
Should we totally remove **ownCloud** here as we have done this in calendar too 😁